lib/coinbase/address/wallet_address.rb

```diff
@@ -39,8 +39,9 @@ def key=(key)
     # @param asset_id [Symbol] The ID of the Asset to send. For Ether, :eth, :gwei, and :wei are supported.
     # @param destination [Wallet | Address | String] The destination of the transfer. If a Wallet, sends to the Wallet's
     #  default address. If a String, interprets it as the address ID.
+    # @param gasless [Boolean] Whether the transfer should be gasless. Defaults to false.
```

Comment:
should we explain what gasless is? "Whether gas fee for the transfer should be covered by Coinbase. This feature is only supported for USDC on Base-Mainnet. Defaults to false."

Reply:
Yeah, I didn't want to make the code documentation be coupled to our backend limitations, namely which assets and networks we support.

E.g. we can add other network support or `:eurc` support without and client-side changes.

Definitely think it makes sense to have a section in our top-level docs indicating what specifically is supported though!